### PR TITLE
CTCP-3240: nav for active mode at border

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

File: app/models/journeyDomain/transportMeans/TransportMeansDomain.scala [hunk 1]
```diff
@@ -22,68 +22,123 @@ import controllers.transportMeans.routes
...
-    InlandModePage.reader.flatMap {
-      case InlandMode.Mail =>
-        UserAnswersReader(TransportMeansDomain(None, None, None))
-      case _ =>
-        (
-          transportMeansDepartureReader,
-          borderModeOfTransportReader,
-          transportMeansActiveReader
-        ).tupled.map((TransportMeansDomain.apply _).tupled)
-    }
```
Comment (jackheslop96) [inline app/models/journeyDomain/transportMeans/TransportMeansDomain.scala:54]: This logic looks to have been wiped out? Has it been moved elsewhere? Or is it no longer needed?

Comment (kenneth-rayner) [reply app/models/journeyDomain/transportMeans/TransportMeansDomain.scala:54]: No longer required. Checked with Matt.  If inland mode method is selected as 'mail' earlier in the journey then these sections are bypassed, It isn't even a radio option on mode crossing border

Comment (jackheslop96) [reply app/models/journeyDomain/transportMeans/TransportMeansDomain.scala:54]: @kenneth-rayner but then if we're not reading that page anywhere it won't get asked?

Comment (kenneth-rayner) [reply app/models/journeyDomain/transportMeans/TransportMeansDomain.scala]: @jackheslop96 we have rectified this

**Pull request closed without merging**